fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/query/RecordQuery.java

```diff
@@ -228,6 +242,17 @@ public Builder setAllowedIndex(@Nullable String allowedIndex) {
             return setAllowedIndexes(Collections.singleton(allowedIndex));
         }
 
+        /**
+         * Set a function that defines whether each index should be used by the query planner.
+         * Note that if allowed indexes are used then the queryability filter is ignored.
```

Comment:
In theory, the docs for `.setAllowedIndexes` should also explain that they override the queryability filter. This also seems like it should link to the `DEFAULT` filter for completeness.

Reply:
Done.